app/models/concerns/course/lesson_plan/item_todo_concern.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+module Course::LessonPlan::ItemTodoConcern
+  extend ActiveSupport::Concern
+
+  included do
+    after_save do |item|
+      if item.try(:actable_type).try(:constantize).try(:has_todo?) && item.draft_changed?
```

Comment:
That's a possible way, but I think you used `constantize` for performance (i.e. no need to load actable object)?

I actually mean just move the `try` thing to an instance method of lesson plan item.